geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/tools/PropertyManagement.java

```diff
@@ -68,46 +77,40 @@ public Object get(
 		return properties.get(toPropertyName(propertyName));
 	}
 
-	public void store(
+	public synchronized void store(
 			final ParameterEnum propertyName,
 			final Object value ) {
-		synchronized (MUTEX) {
-			properties.put(
-					toPropertyName(propertyName),
-					value);
-		}
+		properties.put(
```

Comment:
Minor nit, but probably should be properties.setProperty()
(I think all it does is enforce (String, String), but a good habbit)